bfvmm/src/vmcs/src/vmcs_intel_x64.cpp

```diff
@@ -164,20 +165,7 @@ vmcs_intel_x64::launch()
     // =========================================================================
     // CLEAN ME UP
     // =========================================================================
-
-    // m_memory_manager->alloc_page(&m_msr_bitmap);
-
-    // auto buf = (char *)m_msr_bitmap.virt_addr();
-    // for (auto i = 0; i < m_msr_bitmap.size(); i++)
-    //     buf[i] = 0;
-
-    // vmwrite(VMCS_ADDRESS_OF_MSR_BITMAPS_FULL, (uint64_t)m_msr_bitmap.phys_addr());
-
-    // auto controls = vmread(VMCS_PRIMARY_PROCESSOR_BASED_VM_EXECUTION_CONTROLS);
-
-    // controls |= VM_EXEC_P_PROC_BASED_USE_MSR_BITMAPS;
-
-    // vmwrite(VMCS_PRIMARY_PROCESSOR_BASED_VM_EXECUTION_CONTROLS, controls);
+    vmwrite(VMCS_ADDRESS_OF_MSR_BITMAPS_FULL, (uint64_t)g_mm->virt_to_phys(m_msr_bitmap->address()));
```

Comment:
This needs to be moved to the other VMCS functions were we set the fields that this belongs too